include/h2o/socket/evloop.h

```diff
@@ -42,7 +42,8 @@ typedef struct st_h2o_evloop_t {
         struct st_h2o_evloop_socket_t *head;
         struct st_h2o_evloop_socket_t **tail_ref;
     } _statechanged;
-    uint64_t _now;
+    uint64_t _now_ms;
+    uint64_t _now_ns;
```

Comment:
I'm a bit scared of "ms" and "ns" because they are suspectible to being misread as the other.

Maybe "millisec" and "nanosec"?

Reply:
I see your concern. The point here is clarity so that's even better. Thx!